src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -0,0 +1,164 @@
+= Testing templates
+:page-slug: testing-templates
+:page-description: A consolidated resource for methods a developer can use to test templates
```

Comment:
This description could use a bit more consistency with the rest of the page descriptions. Something like, "Guide to testing Koji templates..." You might find it helpful to do a project search for "page-description" to get a sense of the general flavor and phrasing.  Also, use punctuation (add a period at the end).